include/FxMixer.h

```diff
@@ -73,7 +73,13 @@ class FxChannel : public ThreadableJob
 		void unmuteForSolo();
 
 
-		// TODO C++17 and above: use std::optional insteads
+		void setColor (QColor newColor)
```

Comment:
Now that you have `setColor` for the `FxMixer`, you could use that on the `FxLine` code (`changeColor` and `randomColor`) for consistency.

Reply:
That's a little bit beyond this PR... but since I'm the author of those, I'll do that as well.